nimbus/helm/templates/nimbus-deployment.yaml

```diff
@@ -0,0 +1,95 @@
+{{- range $key, $nimbusClient := .Values.nimbus.clients }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $nimbusClient.name }}
+  namespace: nimbus
+  labels:
+    app: {{ $nimbusClient.name }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ $nimbusClient.name }}
+  replicas: 1
+  strategy: 
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: {{ $nimbusClient.name }}
+    spec:
+      initContainers:
+      - name: init-wait
+        image: alpine:3.13.2
+        command: ['sh', '-c', 'echo Wait for {{ $.Values.nimbus.startWaitTime }} seconds for extra slashing protection! && sleep {{ $.Values.nimbus.startWaitTime }}']
+      containers:
+      - name: nimbus
+        image: "{{ $.Values.image.nimbusImage }}:{{ $.Values.image.versionTag }}"
+        args:
+        {{- if $.Values.ethereumTestnet }}
+        - --network={{ $.Values.ethereumTestnet }}
+        {{- end }}
+        - --non-interactive
+        - --tcp-port={{ $nimbusClient.p2pTcpPort }}
+        - --udp-port={{ $nimbusClient.p2pUdpPort }}
+        - --data-dir=/data/nimbus/beacon
+        - --validators-dir=/data/nimbus/validators
+        - --secrets-dir=/data/nimbus/secrets
+        - --web3-url={{ $nimbusClient.web3Provider }}
+        {{- range .fallbackWeb3Providers }}
+        - --web3-url={{ . }}
+        {{- end }}
```

Comment:
I can see the benefits of this approach is that web3Provider is required field and fallbackWeb3Providers is optional.

The other option is that we turn web3Provider into web3Providers and the variable will take an array instead. The good part about this approach is that we only need to iterate through web3Providers. 

Just want to point out there's another way. I'm fine with either approach.

Reply:
right I prefer to separate them into two different values for now